aries-site/src/examples/components/layouts/FooterExample.js

```diff
@@ -1,14 +1,13 @@
 import React, { useContext } from 'react';
 import { FooterLink } from 'aries-core';
 import { Box, Footer, ResponsiveContext, Text } from 'grommet';
-import { Hpe } from 'grommet-icons';
 
 export const FooterExample = () => {
   const size = useContext(ResponsiveContext);
   return (
-    <Footer
+    <Footer 
       background="background-front"
-      direction={size !== 'small' ? 'row' : 'column'}
+      direction={size !== 'small' ? 'row' : 'column-reverse'}
```

Comment:
So this resulted in the grommet fix for 'column-reverse'(!) AWESOME

Reply:
lol yes!! 👍